File: src/Flashcard.Web/src/app/flashcard-runner/flashcard.ts

export class Flashcard {
    id: string;
    wordToCheck: string;
    wordAnswerTitle: string;
    wordAnswerType: string;
    sentenceFrom: string;
    sentenceTo: string;

    public showAnswer: boolean;

    /**
     *
     */
    constructor() {
        this.showAnswer = false;
    }
}

File: src/Flashcards.Api/Models/FlashCard.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Flashcards.Api.Models
{
    public class Flashcard
    {
        public Guid Id { get; set; }
        public string WordToCheck { get; set; }
        public string WordAnswerTitle { get; set; }
        public string WordAnswerType { get; set; }
        public string SentenceFrom { get; set; }
        public string SentenceTo { get; set; }
    }
}
